### ISSUE TITLE
make install fails with "cannot stat './doxygen/man/man3/*'"

### ISSUE THREAD

**Issue by olysonek** [libevent/libevent#1075]
Hi,

I'm trying to build and install libevent with the html documentation but without man pages. So I'm running the following:
```
./autogen.sh && ./configure --enable-doxygen-doc && make
```
This works just fine, but when I try to install it
```
make DESTDIR=/tmp/tmp.KffdgJYlWT install
```
it fails with the following:
```
 /usr/bin/mkdir -p '/tmp/tmp.KffdgJYlWT/usr/local/share/man/man3'
 /usr/bin/install -c -m 644 './doxygen/man/man3/*' '/tmp/tmp.KffdgJYlWT/usr/local/share/man/man3/*.3'
/usr/bin/install: cannot stat './doxygen/man/man3/*': No such file or directory
make[2]: *** [Makefile:2414: install-man3] Error 1
make[2]: Leaving directory '/home/olysonek/git/libevent'
make[1]: *** [Makefile:3013: install-am] Error 2
make[1]: Leaving directory '/home/olysonek/git/libevent'
make: *** [Makefile:3007: install] Error 2
```

I'm running this on Fedora 31 (but it also fails on Debian) from a clean libevent repo as of commit eb7bed03c4ba38838. The problem also exists in the 2.1.12 release (presumably since f9c6a14ee08945).

If I build the man pages too (add `--enable-doxygen-man` to configure), it installs fine.

I've already spent a day on this and I still have no idea how to fix this. Any ideas? Thanks.

**Comment by azat** [comment]
Thanks for the report! Missing check for was doxygen man pages enabled, should be fixed in the referenced commit

**Comment by olysonek** [comment]
Thanks! This does help me solve the problem.

However there's a caveat to this solution: with the fix in place, a simple `make` no longer builds the documentation - `make doxygen` is necessary. If I do only `make && make install`, I get the following error:
```
/bin/sh /home/olysonek/git/libevent/build-aux/install-sh -d '/tmp/tmp.1041wWGbIL/usr/local/share/doc/libevent'
cp -pR doxygen/html '/tmp/tmp.1041wWGbIL/usr/local/share/doc/libevent/'
cp: cannot stat 'doxygen/html': No such file or directory
cp -pR doxygen/html '/tmp/tmp.1041wWGbIL/usr/local/share/doc/libevent/'
cp: cannot stat 'doxygen/html': No such file or directory
make[2]: *** [Makefile:3444: install-data-local] Error 1
make[2]: Leaving directory '/home/olysonek/git/libevent'
make[1]: *** [Makefile:3013: install-am] Error 2
make[1]: Leaving directory '/home/olysonek/git/libevent'
make: *** [Makefile:3007: install] Error 2
```

**Comment by azat** [comment]
>However there's a caveat to this solution: with the fix in place, a simple make no longer builds the documentation - make doxygen is necessary. If I do only make && make install, I get the following error:

Hm,`make doxygen` should not be requried, can you try clone again and check? (maybe some build cache related)

**Comment by olysonek** [comment]
> Hm,`make doxygen` should not be required, can you try clone again and check? (maybe some build cache related)

I always run `git clean -dfx` before a new build. But I did try cloning again just now. The result is the same.

**Comment by azat** [comment]
>If I do only make && make install, I get the following error:

Fixed in the referenced patch (e3e7bb21)

>I always run git clean -dfx before a new build. But I did try cloning again just now. The result is the same.

Got it, thanks!

**Comment by olysonek** [comment]
> > If I do only make && make install, I get the following error:
> 
> Fixed in the referenced patch ([e3e7bb2](https://github.com/libevent/libevent/commit/e3e7bb212ea17aa8a9d5a30163487342e6ebb350))

That fixed it. Thanks a lot!
